examples/notebooks/scala/taxi_gpu_crossvalidation.ipynb

```diff
@@ -0,0 +1,508 @@
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "val labelColName = \"fare_amount\"\n",
+    "val schema =\n",
+    "    StructType(Array(\n",
+    "      StructField(\"vendor_id\", FloatType),\n",
+    "      StructField(\"passenger_count\", FloatType),\n",
+    "      StructField(\"trip_distance\", FloatType),\n",
+    "      StructField(\"pickup_longitude\", FloatType),\n",
+    "      StructField(\"pickup_latitude\", FloatType),\n",
+    "      StructField(\"rate_code\", FloatType),\n",
+    "      StructField(\"store_and_fwd_flag\", FloatType),\n",
+    "      StructField(\"dropoff_longitude\", FloatType),\n",
+    "      StructField(\"dropoff_latitude\", FloatType),\n",
+    "      StructField(labelColName, FloatType),\n",
+    "      StructField(\"hour\", FloatType),\n",
+    "      StructField(\"year\", IntegerType),\n",
+    "      StructField(\"month\", IntegerType),\n",
+    "      StructField(\"day\", FloatType),\n",
+    "      StructField(\"day_of_week\", FloatType),\n",
+    "      StructField(\"is_weekend\", FloatType)\n",
+    "    ))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a new spark session and load data\n",
+    "we must create a new spark session to continue all spark operations. It will also be used to initilize the `GpuDataReader` which is a data reader powered by GPU.\n",
+    "\n",
+    "NOTE: in this notebook, we have uploaded dependency jars when installing toree kernel. If we don't upload them at installation time, we can also upload in notebook by [%AddJar magic](https://toree.incubator.apache.org/docs/current/user/faq/). However, there's one restriction for `%AddJar`: the jar uploaded can only be available when `AddJar` is called after a new spark session is created. We must use it as below:\n",
+    "\n",
+    "```scala\n",
+    "import org.apache.spark.sql.SparkSession\n",
+    "val spark = SparkSession.builder().appName(\"Taxi-GPU\").getOrCreate\n",
+    "%AddJar file:/data/libs/cudf-0.14-SNAPSHOT-cuda10.jar\n",
+    "%AddJar file:/data/libs/xgboost4j_3.0-1.0.0-SNAPSHOT.jar\n",
+    "%AddJar file:/data/libs/xgboost4j-spark_3.0-1.0.0-SNAPSHOT.jar\n",
+    "%AddJar file:/data/libs/rapids-4-spark-0.1-SNAPSHOT.jar\n",
```

Comment:
Please update these jars.

Reply:
Let's waiting for @wbo4958 for the whole file's status.